.github/workflows/snyk-security.yaml

```diff
@@ -0,0 +1,41 @@
+name: Snyk Security
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  snyk-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+            submodules: recursive
```

Comment:
Is this required?

Reply:
No, good catch! - Changes only need to be made to the `.github/workflows/snyk-security.yaml` file, which is directly accessible in the repo. Will remove, thanks!